docs/runtimes/application.ipynb

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
<p>need to change a few things the example is not up to date</p><p>add the following lines:</p><p>application.set_internal_application_port(&lt;add-your-port&gt;)</p><p>application.spec.command = &lt;add-command-string&gt;</p><p>application.spec.args = &lt;add-command-args-list-of-strings&gt;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6036/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>